packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/host/dedicatedCloud-datacenter-host.controller.js

```diff
@@ -105,6 +106,20 @@ export default class {
       : this.fetchLegacyHostConsumption(hosts);
   }
 
+  fetchLocations(hosts) {
+    return this.$q.all(
+      hosts.map((host) => {
+        return this.dedicatedCloudDataCenterHostService
+          .getHostLocation(this.productId, this.datacenterId, host.hostId)
+          .then((response) => {
+            set(host, 'location', response);
```

Comment:
we prefer not to use lodash any more

Reply:
Didn't know that, was keeping the same coding style as other files, removing it.